### PR TITLE
feat: Toggle columns from LogSidePanel

### PR DESCRIPTION
**Pull request by ernestii**
Allow to toggle columns from LogSidePanel

Screenshot 2023-11-03 at 10 39 13 PM

### CODE REVIEW

**Comment by ernestii** [comment]
oh wait, is changeset supposed to be in root?

File: packages/app/src/LogSidePanel.tsx [hunk 1]
```diff
@@ -1648,6 +1660,24 @@ function PropertySubpanel({
...
+                    as="a"
+                    title={`${
+                      displayedColumns?.includes(keyPathString)
+                        ? 'Remove'
+                        : 'Add'
+                    } ${keyPathString} column`}
+                    style={{ width: 20 }}
+                    onClick={() => toggleColumn(keyPathString)}
+                  >
+                    <i className="bi bi-table" />
```
Comment (ernestii) [inline packages/app/src/LogSidePanel.tsx:1677]: let me know if I should use a different icon :D

File: packages/app/src/LogSidePanel.tsx [hunk 2]
```diff
@@ -1648,6 +1660,24 @@ function PropertySubpanel({
...
+                {!!toggleColumn && keyPath.length === 1 ? (
+                  <Button
+                    className="fs-8 text-muted-hover child-hover-trigger p-0"
+                    variant="link"
+                    as="a"
+                    title={`${
+                      displayedColumns?.includes(keyPathString)
+                        ? 'Remove'
+                        : 'Add'
+                    } ${keyPathString} column`}
```
Comment (MikeShi42) [inline packages/app/src/LogSidePanel.tsx:1673]:
```suggestion
                    } ${keyPathString} column from results table`}
```
Nit: Would make it a tiny bit more clear what the button does, though the table icon should help!

**Review by MikeShi42 — APPROVED** [review]
🚢 lgtm!

**Review by MikeShi42 — APPROVED** [review]

**Pull request merged**